### DISCUSSION TITLE
Connect-PnPOnline Throws error when using hybrid runbook worker

### DISCUSSION THREAD

**Discussion by harshdamaniahd** [pnp/powershell#3886] [General]
Connect-PnPOnline -Url "" -ClientId "" -Tenant "a1ae5425-0bde-496e-8c5a-8a06b0d94277" -Thumbprint "" -ReturnConnection 

Error : [31;1m[0m[36;1m[36;1m[0m[36;1m[0m[36;1m[31;1m[31;1m[36;1m | [31;1mCannot find certificate with this thumbprint in the certificate store.[0m

Pnp Powershell version : 2.2.0
Powershell version : 7.3.6
Hyrbid runbook worker : 7.2

also i see a lot of junk characters

**Comment by AndersRask** [comment]
When using Hybrid runbooks, you are responsible for installing the certificate on the hybrid workers.
Check in certificate store if they are present, else install the same x509 cert that is in the runbook on each hybrid

**Reply by harshdamaniahd** [reply]
The certificate is installed on hybrid worker I.e vm
